### DISCUSSION TITLE
New "Editor Guides" button on level editor menu?

### DISCUSSION THREAD

**Discussion by provigz** [SuperTux/supertux#2075] [Ideas]
This discussion is about #2068. My question is: Do you think it's a good idea to have an "Editor Guides" button on the main level editor menu? It should redirect to the community-made level editor guides, located in the wiki at: [https://github.com/SuperTux/supertux/wiki/Level-Editor](https://github.com/SuperTux/supertux/wiki/Level-Editor). This PR currently doesn't have a dialog for the user to confirm going to that page, but it can easily be added, if having that button is a good idea.

Although it may help people, who are just starting with the editor, there are reasons this idea may not be really good, considering those guides are not really complete and/or are made for an older SuperTux version (most often 0.5.1). As recommended by @Semphriss, let's discuss this here.

**Comment by Rusty-Box** [comment]
I have a Level Editor Guide in the works. There are a few things I yet have to update for the latest version so feel free point out what’s missing.

https://github.com/SuperTux/wiki/pull/30

**Reply by provigz** [reply]
Cool! If this is planned to be finished in recent times, maybe we can just wait, until it gets finished, and then add the button?
